staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/equality.go

```diff
@@ -70,6 +79,35 @@ func getAvoidTimestampEqualities() conversion.Equalities {
 	return avoidTimestampEqualities
 }
 
+func equalIgnoringValueAtPath(a, b any, path []string) bool {
+	if len(path) == 0 { // found the value to ignore
+		return true
+	}
+	aMap, aOk := a.(map[string]any)
+	bMap, bOk := b.(map[string]any)
+	if !aOk || !bOk {
+		return !avoidTimestampEqualities.DeepEqual(a, b)
```

Comment:
surprised at the negation here, comment explaining why or maybe is a typo?

Reply:
Added a comment.

Reply:
Oh oops, you're asking about the call result.. Fixing.